src/components/Button/Button.tsx

```diff
@@ -37,32 +38,41 @@ export enum ButtonTypes {
 
 export type ButtonProps = {
   StyledContainer?: string & StyledComponentBase<any, {}, ButtonContainerProps>;
-  StyledLeftIconContainer?: StyledComponentBase<any, {}>;
-  StyledRightIconContainer?: StyledComponentBase<any, {}>;
+  StyledLeftIconContainer?: StyledSubcomponentType;
+  StyledRightIconContainer?: StyledSubcomponentType;
+
+  skeletonProps?: SubcomponentPropsType;
```

Comment:
Should the size of the skeleton depend on the content in some way? I get the following for the `Button` in the codesandbox, for example:
```typescript
import "./styles.css";

import { Button } from "@headstorm/foundry-react-ui";

export default function App() {
  return (
    <div className="App">
      <h1>Hello CodeSandbox</h1>
      <h2>Start editing to see some magic happen!</h2>
      <Button isLoading>Press me</Button>
    </div>
  );
}
```
![Screen Shot 2021-08-09 at 10 52 06 AM](https://user-images.githubusercontent.com/10625031/128736026-43fb24e8-be0f-4f71-a32e-b965caee84b7.png)
Or is it intended for the user to specify things like a max width in the `skeletonProps`?

Reply:
Hmm the display CSS rule on the Button's loading is supposed to be `inline-block`.. not sure when I messed that up. good catch!
![image](https://user-images.githubusercontent.com/7891559/128760623-2bb5027a-7ec2-49d1-b414-67cc7625af36.png)